src/main/scala/codacy/codesniffer/docsgen/CategoriesMapper.scala

```diff
@@ -1,26 +1,33 @@
 package codacy.codesniffer.docsgen
 import codacy.codesniffer.docsgen.parsers.PatternIdParts
 import com.codacy.plugins.api.results.Pattern
-import com.codacy.plugins.api.results.Pattern.Category
+import com.codacy.plugins.api.results.Pattern.{Category, Subcategory}
 
 object CategoriesMapper {
 
-  private[this] val manualCategories: Map[String, Pattern.Category.Value] =
-    Map("WordPress_DB_PreparedSQL" -> Category.Security,
-        "WordPress_WP_PreparedSQL" -> Category.Security,
-        "WordPress_XSS_EscapeOutput" -> Category.Security)
+  def categoryFor(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Cyclomatic complexity of 18 exceeds max of 10](https://app.codacy.com/gh/codacy/codacy-codesniffer/pullRequest?prid=4955957)